action.yml

```diff
@@ -9,6 +9,9 @@ inputs:
     required: false
   namespace:
     description: "Relevant if renderEngine == helm. Namespace to be used for Helm option."
+  arguments:
+    description: "Relevant if renderEngine == helm. Array of arguments. Each argument should be mentioned on a newline"
```

Comment:
It's not an array. Specify that it's going to be passed to the eventual helm command